packages/preview/leipzig-glossing/0.3.0/typst.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "leipzig-glossing"
+version = "0.3.0"
+entrypoint = "leipzig-gloss.typ"
+authors = ["Greg Shuflin <greg@everydayimshuflin.com>", "Other open-source contributors"]
+license = "MIT"
+description = "Linguistic interlinear glosses according to the Leipzig Glossing rules"
+repository = "https://code.everydayimshuflin.com/greg/typst-lepizig-glossing"
+disciplines = ["linguistics"]
+categories = ["paper"]
+keywords = ["linguistics", "leipzig", "gloss", "glossing"]
+compiler = "0.11"
```

This should be a full version.

```suggestion
compiler = "0.11.0"
```